BuildAndTest.cmd

```diff
@@ -59,6 +59,9 @@ echo Creating Platform Specific BinSkim 'Publish' Packages
 call :CreatePublishPackage netcoreapp3.1 win-x64 || goto :ExitFailed
```

Comment:
>CreatePublishPackage [](http://example.com/codeflow?start=6&length=20)

Is this right? Why don't we have a single CreatePublishPackage step for each platform?

We don't create a netcoreapp3.1 specific package and also a net6.0 specific package, we should be creating a single package with both frameworks.

It's possible our script isn't set up to do this.

Let's get this change in and take a further look at my question.